examples/binaries/calc-hash/calculator/src/code.rs

```diff
@@ -0,0 +1,11 @@
+use gstd::{msg, prelude::*};
```

Comment:
btw, this demo may be a part of aggregator itself (special command from source == self id)
or may contain more that one calculation iteration to reduce amount of messages sendings